lib/flipper/adapters/active_record.rb

```diff
@@ -278,6 +278,9 @@ def result_for_gates(feature, gates)
       # See https://github.com/flippercloud/flipper/pull/692
       def value_not_text?
         @gate_class.column_for_attribute(:value).type != :text
+      rescue ::ActiveRecord::StatementInvalid => error
```

Comment:
We're also seeing `ActiveRecord::ConnectionNotEstablished` when building a docker image. 

(I changed my initializer to just blow off all `ActiveRecord::AdapterError`.)